docs($log): describe your change...

davesidious opened the pull request:
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Document update

**What is the current behavior? (You can also link to an open issue here)**

The documentation for $log doesn't mention blackboxing, by which a developer can see the originating line of calls to $log.

**What is the new behavior (if this is a feature change)?**

A brief, browser-agnostic mention of blackboxing was added to the page.

**Does this PR introduce a breaking change?**

No.

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

gkalpak commented on src/ng/log.js:
```diff
@@ -11,6 +11,9 @@
  * The main purpose of this service is to simplify debugging and troubleshooting.
  *
+ * To reveal the location of the calls to `$log` in the JavaScript console,
+ * "blackbox" the Angular source in your browser.
```
I believe this will be confusing for people that don't know what `blackboxing` means (in this context). I would at least add a link to an explanation of blackboxing.

davesidious replied:
That is a good idea.  I've looked and I can't find a neutral source of sufficient standing to cite.  The closest are the documentation pages from Chrome and Firefox which outline how to set blackboxing up.  I guess we can just drop this PR if one of these links wouldn't suffice:

https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Black_box_a_source
https://developer.chrome.com/devtools/docs/blackboxing

Narretz replied:
I think these links are okay. You can add both and a note that *some* browsers supprt blackboxing.

Narretz commented:
@davesidious are you still interested in making the required changes to this PR?

Commit:
docs($log): Add blackboxing hint

Add browser-agnostic hint about blackboxing and the benefits it brings developers when using $log.

davesidious commented:
@Narretz I have pushed an update to the patch-5 branch.  Sorry for letting this one slip through!

frederikprijck commented on src/ng/log.js:
```diff
@@ -11,6 +11,14 @@
  * The main purpose of this service is to simplify debugging and troubleshooting.
  *
+ * To reveal the location of the calls to `$log` in the JavaScript console,
+ * "blackbox" the Angular source in your browser.
```
Angular => AngularJS 🙈

Narretz committed:
typos + style
